OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -370,6 +370,9 @@ public void AddInfluence(Actor self, IOccupySpace ios)
 				if (cellTriggerInfluence.TryGetValue(c.First, out triggers))
 					foreach (var t in triggers)
 						t.Dirty = true;
+
+				if (CellUpdated != null)
+					CellUpdated(c.First);
```

Comment:
Thinking out loud, a discussion point rather than a change request: Doing this cell-by cell means that each implementation will need to maintain its own dirty cache. Would it make more sense to keep a single dirty list here and then call `Action<IEnumerable<CPos>> CellsUpdated` once from `Tick`?

Reply:
Where and when do we clear this dirty list?

Reply:
immediately after notifying `CellsUpdated`